redash/tasks/general.py

```diff
@@ -85,8 +85,8 @@ def get_schema(data_source_id, refresh):
                 "message": "Data source type does not support retrieving schema",
             }
         }
-    except Exception:
-        return {"error": {"code": 2, "message": "Error retrieving schema."}}
+    except Exception as e:
+        return {"error": {"code": 2, "message": f"Error retrieving schema: {e}"}}
```

Comment:
How about we split it into a separate field (`details`?). This way we can better format it in the UI.

Reply:
Oh yes, this is a good idea. Let me adjust it here. Thank you for review!